operator/scripts/retry.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+# shellcheck source=./hack/common.sh
+source "$(dirname "$0")/../hack/common.sh"
+
+eecho() {
```

Comment:
note `common.sh` also introduces `log` which has a bit more functionality and a name that does not look like a typo 😉

Reply:
done